network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgWriter.java

```diff
@@ -899,20 +899,21 @@ private void addMetadata(Graph graph, XMLStreamWriter writer) throws XMLStreamEx
                 busNode.getEquipmentId(),
                 String.valueOf(busNode.getNbNeighbouringBusNodes()),
                 String.valueOf(busNode.getIndex()),
-                getPrefixedId(vlNode.getDiagramId())
-        )));
+                getPrefixedId(vlNode.getDiagramId()))));
         graph.getNodesStream().forEach(node -> metadata.addNode(getPrefixedId(node.getDiagramId()), node.getEquipmentId(),
                 getFormattedValue(node.getX()), getFormattedValue(node.getY())));
         graph.getBranchEdgeStream().forEach(edge -> metadata.addEdge(getPrefixedId(edge.getDiagramId()), edge.getEquipmentId(),
                 getPrefixedId(graph.getNode1(edge).getDiagramId()),
                 getPrefixedId(graph.getNode2(edge).getDiagramId()),
                 getPrefixedId(graph.getBusGraphNode1(edge).getDiagramId()),
-                getPrefixedId(graph.getBusGraphNode2(edge).getDiagramId())));
+                getPrefixedId(graph.getBusGraphNode2(edge).getDiagramId()),
+                edge.getType()));
         graph.getThreeWtEdgesStream().forEach(edge -> metadata.addEdge(getPrefixedId(edge.getDiagramId()), edge.getEquipmentId(),
                 getPrefixedId(graph.getNode1(edge).getDiagramId()),
                 getPrefixedId(graph.getNode2(edge).getDiagramId()),
                 getPrefixedId(graph.getBusGraphNode1(edge).getDiagramId()),
-                getPrefixedId(graph.getBusGraphNode2(edge).getDiagramId())));
+                getPrefixedId(graph.getBusGraphNode2(edge).getDiagramId()),
+                "ThreeWtEdge"));
```

Comment:
It would be cleaner if `ThreeWtEdge` holds that string type.

In fact we should move the type attribute which is in `BranchEdge` in `AbstractEdge`, and add `getType()` method in the `Edge` empty interface. Could you do such a change in this PR?